miio/integrations/petfeeder/status.py

```diff
@@ -0,0 +1,85 @@
+from datetime import timedelta
+from enum import Enum
+from typing import Any, Dict
+
+from miio import DeviceStatus
+
+
+class FoodLevel(Enum):
+    Normal = 0
+    Low = 1
+    Empty = 2
+
+
+class FoodOutletStatus(Enum):
+    Ok = 0
+    Blocked = 1
+
+
+class FoodStorageLid(Enum):
+    Closed = 0
+    Open = 1
+
+
+class PetFeederStatus(DeviceStatus):
+    """Container for status reports from the Pet Feeder / Smart Pet Food Dispenser."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        """
+        Response from pet feeder (mmgg.feeder.petfeeder):
+
+        {
+            "food_level": 0,
+            "feed_plan": 1,
+            "food_outlet": 0,
+            "undeclared": 0,
+            "clean_days_left": 11,
+            "food_storage_lid": 0,
+            "desiccant_days_left": 21,
+            "weight_level": 0,
+            "wifi_led": 0,
+            "button_lock": 1,
+            "country_code": 255
+        }
+        """
+        self.data = data
+
+    @property
+    def food_level(self) -> FoodLevel:
+        """Current food level / status."""
+        return FoodLevel(self.data["food_level"])
+
+    @property
+    def feed_plan(self) -> bool:
```

Comment:
`automated_feeding_enabled`?

Reply:
I went forward and backward on this a few times.
The _WHOLE_ schedule system is referred to as a feed(ing) plan. I'm erring on the side of the person who owns the device should kinda know what it does and be obvious to them, especially since I'm using similar phrasing as MiHome where appropriate.

Reply:
The docstring talks about "automatic feeding status", so it feels weird if the name is something else. Even `automatic_feeding_status -> bool` would give any reader more info as the current name... :-P